src/azul/transformer.py

```diff
@@ -141,6 +141,19 @@ def translate_fields(cls, source: AnyJSON, path: Tuple[str, ...] = (), forward:
         else:
             return cls.translate_field(source, path=path, forward=forward)
 
+    @classmethod
+    def untranslate_aggregates(cls, es_response):
+        """
+        Untranslate aggregated field values using the ES field name stored in the 'meta' field
+        """
+        for aggregation in es_response.aggregations:
+            if 'myTerms' in aggregation:
```

Comment:
Why?

Reply:
It is necessary to specify the 'myTerms' property by name here due to the class BucketData having an iter function that iterates over the object's 'bucket' property only.

Reply:
Hmm, I doubt that there is no less brittle way to iterate over the aggregation. Remember how we  assumed we had to use protected members which turned out to be false upon closer inspection of the code? If you really think you can't work around this, please prepare a debugging session during which you and I can verify together if what you assert is correct.

Reply:
Digging through the class hierarchy I was able to find `dir()` can be used to get a list of keys